### ISSUE TITLE
Email verification link is not working

### ISSUE THREAD

**Issue by liveHarshit** [fossasia/open-event-server#5969]
![image](https://user-images.githubusercontent.com/27783786/58566870-fd492500-824e-11e9-963e-3ccb84f48b26.png)

**Comment by mrsaicharan1** [comment]
@liveHarshit Can you point me to the exact link which is being redirected to?

**Comment by liveHarshit** [comment]
@mrsaicharan1 Email is - 
```
Hi live.harshit@hotmail.com,
Please visit this link to confirm your email: https://fossasia.github.io/open-event-frontend/verify?token=WyJsaXZlLmhhcnNoaXRAaG90bWFpbC5jb20iLCAiWFBPUlo5Il0uc3ljNlN0VElYRU10SV80SGtDV2oxRk5kQV9n
```

**Comment by kushthedude** [comment]
> @mrsaicharan1 Email is -
> 
> ```
> Hi live.harshit@hotmail.com,
> Please visit this link to confirm your email: https://fossasia.github.io/open-event-frontend/verify?token=WyJsaXZlLmhhcnNoaXRAaG90bWFpbC5jb20iLCAiWFBPUlo5Il0uc3ljNlN0VElYRU10SV80SGtDV2oxRk5kQV9n
> ```

@liveHarshit The following deployment you are using is of `gh-Pages` , Please check on master or `eventyay`

**Comment by iamareebjamal** [comment]
@kushthedude The link he's using is correct because the backend he's using is open-event-dev

**Comment by liveHarshit** [comment]
> @kushthedude The link he's using is correct because the backend he's using is open-event-dev heroku app

Yes, I want to create test events so I use the debug build. But my account is not verified. 
@iamareebjamal Should we use the same in attendee app that user cannot buy tickets without verifying the email?

**Comment by liveHarshit** [comment]
> Should we use the same in attendee app that user cannot buy tickets without verifying the email?

Fund discussion about it. https://github.com/fossasia/open-event-server/issues/5760 :+1:

**Comment by uds5501** [comment]
@liveHarshit @iamareebjamal   This issue will be fixed in fossasia/open-event-frontend#3041

**Comment by iamareebjamal** [comment]
@uds5501 This issue is unrelated to fossasia/open-event-frontend#3041

A user should nevertheless be able to verify his/her account
